src/components/map/toolbar/ToolbarCommonButtons.tsx

```diff
@@ -46,11 +50,19 @@ class ToolbarCommonButtons extends React.Component {
                     <MapControlButton
                         onClick={this.undo}
                         isActive={false}
-                        isDisabled={!GeometryEventStore.hasEvents}
-                        label='Undo'
+                        isDisabled={false}
+                        label='Kumoa'
                     >
                         <IoMdUndo />
                     </MapControlButton>
+                    <MapControlButton
+                        onClick={this.redo}
+                        isActive={false}
+                        isDisabled={false}
+                        label='Tee uudestaan'
```

Comment:
'Tee uudestaan'? Is that actually the correct term in finnish?

Reply:
Didn't figure out anything better. Found it from here: https://www.sanakirja.org/search.php?q=redo&l=3&l2=17